src/GitHub.Exports/Exports/ExportMetadata.cs

```diff
@@ -69,7 +69,7 @@ public ExportMenuAttribute() : base(typeof(IMenuHandler))
     public interface IViewModelMetadata
     {
         [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
-        Type[] ViewModelType { get; }
+        string[] ViewModelType { get; }
```

Comment:
Why did you change the type of this member? It's not mentioned in the PR description.

Reply:
My aim was to remove `ViewViewModelFactory`s dependency on `IGitHubServiceProvider`.

Instead of calling `IGitHubServiceProvider.ExportProvider.GetExport<TViewModel>()`, I was hoping to use a second `ExportFactory`, like so:

```
        [ImportMany]
        IEnumerable<ExportFactory<FrameworkElement, IViewModelMetadata>> Views { get; set; }

        [ImportMany("ViewModel")]
        IEnumerable<ExportFactory<object, IViewModelMetadata>> ViewModels { get; set; }
```

We would then enumerate through the `IViewModelMetadata` when finding a factory to create a View or ViewModel.  If the `IViewModelMetadata` contained a `Type` rather than a `string`, it would force *all* assemblies to load the first time a View or ViewModel was created (hence the change to using `string`).

Unfortunately there appear to be some strange limitations with `ExportFactory` which prevent this from working. 😢

Reply:
I think the change from `Type` to `string` is still valuable because it should prevent all assemblies that contain a view from being pre-loaded when the first view is created.  Does that make sense?

Reply:
Yes, that makes sense, need to check that this is actually happening!